includes/class-wc-subscriptions-core-plugin.php

```diff
@@ -153,11 +153,9 @@ public function init() {
 		// Initialise the cache.
 		$this->cache = WCS_Cache_Manager::get_instance();
 
-		if ( class_exists( 'Automattic\WooCommerce\Blocks\Package' ) && version_compare( \Automattic\WooCommerce\Blocks\Package::get_version(), '4.4.0', '>' ) ) {
-			// When WooCommerceBlocks is loaded, set up the Integration class.
-			add_action( 'woocommerce_blocks_loaded', array( $this, 'setup_blocks_integration' ) );
-			add_action( 'woocommerce_blocks_loaded', array( 'WC_Subscriptions_Extend_Store_Endpoint', 'init' ) );
-		}
+		// When WooCommerceBlocks is loaded, set up the Integration class.
+		add_action( 'woocommerce_blocks_loaded', array( $this, 'setup_blocks_integration' ) );
```

Comment:
I thought about consolidating these callbacks to just the one: `setup_blocks_integration()`

Then inside that function manually call: `WC_Subscriptions_Extend_Store_Endpoint::init()`

It breaks backwards compatibility but curious to get your thoughts!